demo-app/src/main/java/com/gravatar/demoapp/ui/DemoGravatarApp.kt

```diff
@@ -151,7 +149,7 @@ private fun GravatarTabs(
         stringResource(R.string.tab_label_avatar_update),
     )
 
-    Column(modifier = Modifier.fillMaxSize()) {
+    Column(modifier = modifier.fillMaxSize()) {
```

Comment:
I noticed this didn't work well after switching to 35. We should always pass the modifier to the Top level composable, not its children.